google-ads/src/test/resources/com/google/ads/googleads/lib/mock_service_account_secrets.json

```diff
@@ -0,0 +1,12 @@
+{
+  "type": "service_account",
+  "project_id": "test-service-account",
+  "private_key_id": "test-private-key-id",
+  "private_key": "-----BEGIN PRIVATE KEY-----\nMIIBVgIBADANBgkqhkiG9w0BAQEFAASCAUAwggE8AgEAAkEAtBajzOZvaWZrQyy9IT2uTdOHOs3WboTGnnyoZW6q3CFpfCm4U8LKxCR28aO/CBSpg8DnBTm/TfM+duY72Pk69wIDAQABAkEAo2Gaz60LzMLHc1UmbP7poN/UYswVy4SYMXcEy6lAR55b/ccd3WLFcCPo+UrgWBQB0s76jjodPhNCLmNBK5D2CQIhAPJq9umTVG3IoUkdFk0ovCtcIij1BS+ypQzQMLl8RlpbAiEAvi2s6ro+17ttBqtwTTJ/tGF1YGxrpBINeZHYqqetLJUCIBElBX9xfwggp19/YiNk3SIH6xaQdaRw8wFb/AYyRtt/AiEAg02F1YpBWTFOVQyIvMPg5+lh4PD6YI0WQJkxvqGGLwECIQC4AO8/yGJQsBd0tXcJi90MFZMyTjK9IpzksA9/LQF+VA==\n-----END PRIVATE KEY-----\n",
```

Comment:
This looks like a real account key. Do you really want to check it in?

Reply:
Thanks for checking. I realize it looks like a real key, but it isn't. I generated it just for this test because `ServiceAccountCredentials.fromStream()` requires a file with a private key that adheres to the spec.

Reply:
See also [my previous comment](https://github.com/googleads/google-ads-java/pull/297#discussion_r528792920) (buried in the GitHub convo).